sdk/model/tag_object.go

```diff
@@ -0,0 +1,15 @@
+/*
+ * Orkes Conductor API Server
+ *
+ * Orkes Conductor API Server
+ *
+ * API version: v2
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package model
+
+type TagObject struct {
```

Comment:
can we find a better name?

Reply:
Again, auto generated. But maybe I can create a wrapper which uses this in the API.

Reply:
Added a new class called MetadataTag which needs to be passed to the new methods now.